### PR TITLE
Taql leak fixing and measures support, second attempt

### PR DESCRIPTION
**Pull request by tammojan**
PR #734 was reverted in #747 because it broke the CASA build. A few minor changes in CASA make it possible to compile CASA with and without the change in this PR.

@mpokorny: could you make the changes in CASA and then merge this PR?

### CODE REVIEW

File: tables/TaQL/ExprNodeSet.h [hunk 1]
```diff
@@ -365,7 +370,7 @@ class TableExprNodeSet : public TableExprNodeRep
     Bool isBounded() const;
 
     // Get the number of elements.
-    uInt nelements() const;
+    uInt size() const;
```
Comment (mpokorny) [inline tables/TaQL/ExprNodeSet.h]: Can we keep nelements() as an alias? That would of simplify integrating these changes with CASA.

**Comment by mpokorny** [comment]
I've built my CASA branch successfully with this casacore branch and with the casacore master branch. I'm currently waiting for the CASA branch to be merged to master. I'll be on vacation next week, but I've left instructions with @vsuorant on the CASA team, and have included @gervandiepen in the conversation on the CASA JIRA site, about how to proceed.

**Comment by vsuorant** [comment]
CASA PR is submitted and will likely be merged tomorrow.

**Comment by vsuorant** [comment]
Yes. This can be merged.

**Pull request merged**